lib/cassandra/cluster/schema/fetchers.rb

```diff
@@ -1361,7 +1361,7 @@ def create_table_options(table_data, compaction_strategy, is_compact)
           def create_column(column_data, types)
             name      = column_data['column_name']
             is_static = column_data['kind'].to_s.casecmp('STATIC').zero?
-            order     = column_data['clustering_order'] == 'desc' ? :desc : :asc
+            order     = column_data['clustering_order'].to_s.casecmp('DESC').zero? ? :desc : :asc
```

Comment:
Why not use `casecmp?` ? It would allow to write it like this:
```ruby
column_data['clustering_order'].to_s.casecmp?('DESC') ? :desc : :asc
```

Reply:
@kapcod Sure! I wrote it like that just to be consistent with the style of the line above which uses `casecmp('STATIC').zero?`. But I can change it with `casecmp?` (which is more straightforward) if you prefer.